salt/common/tools/sbin/so-status

```diff
@@ -15,21 +15,23 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-if ! [ "$(id -u)" = 0 ]; then
-   echo "This command must be run as root"
-   exit 1
-fi
+MYNAME="${0##*/}"
```

Comment:
This breaks standard from the other Security Onion scripts, which use the full path (i.e. `$0`).

Reply:
I'd still like to capture the value of `$0` at script start since there's no guarantee that `$0` wont change. We can still reference the scrip's full path with:

```
$(readlink -f ${MYPATH}/${MYNAME})
```

Or something like that.

Reply:
This script isn't doing anything that might change `$0`, and the value is being used solely by the usage function `display_help()` and the root check. There's no need to capture its value.

However,  I was conflating the full path and the value of `$0` so referencing the full path isn't actually necessary either.

Reply:
The value of `$0` is shell dependent so silly things can happen, but it's been changed back.

Reply:
In this case we don't need to worry about other shells since the shebang specifies this runs under bash.